libraries/app/include/scorum/app/betting_api_impl.hpp

```diff
@@ -0,0 +1,103 @@
+#include <scorum/app/betting_api.hpp>
+
+#include <chainbase/database_guard.hpp>
+#include <scorum/chain/database/database.hpp>
+
+#include <scorum/chain/services/bet.hpp>
+#include <scorum/chain/services/pending_bet.hpp>
+#include <scorum/chain/services/matched_bet.hpp>
+#include <scorum/chain/services/game.hpp>
+
+#include <scorum/app/application.hpp>
+
+#include <boost/range/adaptor/transformed.hpp>
+#include <boost/range/algorithm/transform.hpp>
+#include <boost/range/algorithm_ext/copy_n.hpp>
+#include <boost/range/adaptor/filtered.hpp>
+
+#include <scorum/utils/range_adaptors.hpp>
+
+#include <scorum/utils/take_n_range_adaptor.hpp>
+
+namespace scorum {
+namespace app {
+
+#define BETTING_API_MAX_QUERY_LIMIT 1000
```

Comment:
We have common approach for such a limit. See `get_api_config` function. It allows to configure limits via config